### PR TITLE
Search engine2

### PR DESCRIPTION
**Pull request by omernu**

### CODE REVIEW

**Review by simzacks — CHANGES_REQUESTED** [review]
When modifying an existing PR (52), you should use git commit --amend. That means it is the same commit.  When you push it tou your github, use the same branch on your github that you originally pushed to. That will automatically update the existing PR

File: search_engine.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+def select_preferences_by_UserID(conn, User):
+    """
+    Query preferences by UserID
+    :param conn: the Connection object
+    :param UserID:
+    :return:
+    """
+    cur = conn.cursor()
+    cur.execute("SELECT * FROM Users WHERE User=?", (User,))
```
Comment (Yarboa) [inline search_engine.py:29]: Do you have Users table? while i uploaded the Vagrant files, i did not see there is Users table,
BTW django is not up, please rebase Vagrant file.
If you have new Users Table different than djando user_authentication

File: search_engine.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    # create a database connection
+    conn = create_connection(database)
+    User = none
+    with conn:
+        select_preferences_by_UserID(conn, User)
+      
+
+if __name__ == '__main__':
+    main()
```
Comment (Yarboa) [inline search_engine.py:51]: See comment new line is missing under main

**Review by Yarboa — COMMENTED** [review]
I can advise you to install sqlite client, to understand what you have inside db https://www.sqlitetutorial.net/sqlite-commands/

**Note:** Please make a habit run flake fevore git push

**Pull request closed without merging**